R/uvhydrograph-render.R

```diff
@@ -344,10 +344,18 @@ createSecondaryPlot <- function(uvInfo, secondarySeriesList,
   
   if(!isEmptyOrBlank(secondarySeriesList[['corrected']][['points']][['value']])){
     lims <- calculateLims(secondarySeriesList[['corrected']][['points']])
-    corrLimPoints <- secondarySeriesList[['corrected']][['points']][['value']]
```

Comment:
Hmmm I think all you need is to have estimated and corrected grouped together before the limits are calculated. Then you don't need to add anything.

```
allCorrectedData <- rbind(secondarySeriesList[['corrected']][['points']], secondarySeriesList[['estimated']][['points']])
if(!isEmptyOrBlank(allCorrectedData )){
    lims <- calculateLims(allCorrectedData )
    lims[['ylim']] <- bufferLims(primaryLims[['ylim']], secondarySeriesList[['uncorrected']][['points']][['value']])
  } else {
    lims <- calculateLims(secondarySeriesList[['uncorrected']][['points']])
  }
```

^ this copies what primary plot does. Since estimated would be used no matter what, the additional scenario for "useEstimated" would not be necessary.